test/FastTests/Corax/SimplePipeline.cs

```diff
@@ -2,36 +2,37 @@
 using System.Text;
 using Corax.Analyzers;
 using Corax.Pipeline;
+using Corax.Pipeline.Parsing;
 using FastTests.Voron;
+using Sparrow;
+using Tests.Infrastructure;
 using Xunit;
 using Xunit.Abstractions;
 
 namespace FastTests.Corax
 {
 
-    public class SimplePipelineTest : StorageTest
+    public class SimplePipelineTest(ITestOutputHelper output) : StorageTest(output)
     {
-        public SimplePipelineTest(ITestOutputHelper output) : base(output) { }
-
-        [Fact]
```

Comment:
After changing that you need to adjust `AllTestsShouldUseRavenFactOrRavenTheoryAttributes` test.